app/indexers/sorted_date_indexer.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+##
+# A mixin for Valkyrie resources to support Date Range facet.
+module SortedDateIndexer
+  # rubocop:disable Metrics/BlockLength
+  [:to_solr].each do |method_name|
```

Comment:
Not a blocker, but there's no reason to do it this way... this was done so one module would work for both `generate_solr_document` and `to_solr`, to share it between resources and works. a simple `def to_solr` is much more straightforward.